tv_apps/app-deck/js/app_deck.js

```diff
@@ -129,7 +109,7 @@
             pinned: pinned,
             manifestURL: elem.dataset.manifestURL,
             entryPoint: elem.dataset.entryPoint,
-            name: elem.dataset.name
```

Comment:
I would suggest to keep using `dataset.name` because it should be faster than getAttribute();